src/core/genericInterpreters/lastFallback.ts

```diff
@@ -134,20 +133,21 @@ function lastFallback(decodedData: DecodedTx, interpretation: Interpretation) {
 
     const symbolReceived = tokensReceived[0]?.symbol || chainSymbol
 
```

Comment:
Could do `const action = interpretation.action[0]` and then use the `action` variable throughout the rest

Reply:
same for in `token.ts` and `transfer.ts`